File: intro/spring/server/jsf/src/main/java/org/tsdes/intro/spring/jsf/Application.java
package org.tsdes.intro.spring.jsf;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Created by arcuri82 on 12-Dec-17.
 */
@SpringBootApplication
public class Application {

    /**
     * In a "packaged" Java application, this method would be the entry point.
     * This is also useful to have for debugging, as in
     * an IDE you can just right-click on this class to start
     * the whole application.
     */
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: advanced/front-end/websocket/src/main/kotlin/org/tsdes/spring/frontend/websocket/Application.kt
package org.tsdes.spring.frontend.websocket

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * Created by arcuri82 on 16-Nov-17.
 */
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}
